packages/agoric-cli/lib/main.js

```diff
@@ -218,6 +221,22 @@ const main = async (progname, rawArgs, powers) => {
       return subMain(startMain, ['start', profile, ...args], opts);
     });
 
+  program
+    .command('update')
+    .description('Update the Agoric SDK.')
+    .action(async () => {
+      /**
+       * Change to Agoric SDK in user's home directory.
+       */
+      chdir(resolve(homedir(), 'agoric-sdk'));
+      /**
+       * Git pull, install, and build.
+       */
+      spawnSync('git', ['pull', '--depth', '1'], { stdio: 'inherit' });
```

Comment:
The other changes I'd suggest making are using an async spawn instead, and verifying that the `--sdk` program argument was passed in (otherwise the agoric CLI is running from something like `npx`, not a linked `agoric-sdk`).